pkg/v2/tkr/webhook/infra-machine/Makefile

```diff
@@ -43,7 +43,11 @@ docker-publish: ## Publish docker image
 
 .PHONY: kbld-image-replace
 kbld-image-replace: ## Add newImage in kbld-config.yaml
-	cd ../../../../../hack/packages/kbld-image-replace && $(MAKE) run IMAGE=$(IMG_DEFAULT_NAME_TAG) NEW_IMAGE=$(IMG)
+	cd ../../../../../hack/packages/kbld-image-replace && \
+		go run main.go -kbld-config ../../../packages/management/tkr-aws-machine-webhook/kbld-config.yaml $(IMG_DEFAULT_NAME_TAG) $(IMG) && \
```

Comment:
can we cache the binary and reuse it, instead of building it each time?

Reply:
I'm unsure what you mean by caching the binary. We could generate a kbld-config once and copy it to each `tkr-*-machine-webhook` folder. Is that what you had in mind?

Reply:
Per our discussion I think we'll optimize/refactor this in another PR.